### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by sanuann**

### CODE REVIEW

File: docs/pages/schema_props.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+| reproterms:shuffle       | An element (bool) to determine if the list of items is shuffled or in order.                                                                                                                                                                                                 | reproschema:Activity | schema:boolean                         |
+| reproterms:addProperties | An array of objects to describe the various properties added to assessments and fields.                                                                                                                                                                                      | reproschema:Activity | reproschema:AddProperties              |
+| reproterms:scoringLogic  | An array of objects indicating score in an activity or protocol and the score Field item                                                                                                                                                                                     | reproschema:Activity | reproschema:ScoringLogic               |
+# Properties of Field schema
```
Comment (Remi-Gau) [inline docs/pages/schema_props.md:27]: Am I missing something or shouldn't the next section give details about  the Item schema ?

To me it looks like this is a duplicate of the schema of activity.

**Comment by Remi-Gau** [comment]
@sanuann FYI: (finally) working on a PR to send on your branch to start updating the index.md of the doc.

File: docs/pages/schema_props.md (hunk 1 above)
Comment (sanuann) [reply docs/pages/schema_props.md:27]: oops! Sorry I will update that.

**Comment by satra** [comment]
@sanuann - i think this PR should be closed. it's superceded by #399

**Pull request closed without merging**